src/silo_api/query_handler.cpp

```diff
@@ -50,9 +50,11 @@ void QueryHandler::post(
             );
          }
       }
-   } catch (const silo::QueryParseException& ex) {
+   } catch (const silo::QueryException& ex) {
       response.setContentType("application/json");
-      SPDLOG_INFO("Query is invalid: " + query + " - exception: " + ex.what());
+      SPDLOG_INFO(
+         "Query is invalid: {} - exception during {}: ", query, ex.duringString(), ex.what()
```

Comment:
```suggestion
         "Query is invalid: {} - exception during {}: {}", query, ex.duringString(), ex.what()
```

Reply:
Oh, no compile time check. (Another reason to use Rust.) Good catch!

Reply:
I was wondering why the log statement didn't include the message... :D